functions/Install-DbaWhoIsActive.ps1

```diff
@@ -186,7 +190,19 @@ function Install-DbaWhoIsActive {
 						Stop-Function -Message "Failed to install stored procedure." -ErrorRecord $_ -Continue -Target $instance
 					}
 				}
-				
+				$baseres = @{
+					ComputerName = $server.NetName
+					InstanceName = $server.ServiceName
+					SqlInstance  = $server.DomainInstanceName
+					Database     = $Database
+					Name         = $scriptname.TrimEnd('.sql')
+				}
+				if('sp_WhoisActive' -in $allprocedures) {
```

Comment:
This should be [pscustomobject]@ right out the gate without assigning to $baseres first.

Then what you can do is decide the status prior to creating the object, and then assign that status

so 

````
if('sp_WhoisActive' -in $allprocedures) {
$status = "Updated"
````

Then here:

````
[pscustomobject]@{
ComputerName = $server.NetName
InstanceName = $server.ServiceName
SqlInstance  = $server.DomainInstanceName
Database     = $Database
Name         = $scriptname.TrimEnd('.sql')
Status = $status
}
````

thanks baw!